lightpath/controller.py

```diff
@@ -1,12 +1,20 @@
 """
 While the :class:`.BeamPath` object provides basic control functionality, the
 :class:`.LightController` is what does the organization of all of LCLS's
-devices. After parsing through all of the given devices, each beamline is
-contsructed as a :class:`.BeamPath` object. This includes not only devices on
-the upstream beamline but all of the beamlines before it. For example, the MEC
-beampath will include devices in both the FEE and the XRT. The
-:class:`.LightController` handles this logic as well as a basic overview of
-where the beam is and what the state of the MPS system is currently.
+devices. The facility is represented by an Direced Graph, starting at the
+source and ending at the beamline hutches.  After parsing through all of the
+given devices, each beamline is contsructed as a :class:`.BeamPath` object.
```

Comment:
```suggestion
given devices, each beamline is constructed as a :class:`.BeamPath` object.
```

Also, `parsing` (above) might not be the most appropriate word if I understand the section correctly